cirq/google/api/v2/batch.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+import "cirq/google/api/v2/program.proto";
+import "cirq/google/api/v2/result.proto";
+import "cirq/google/api/v2/run_context.proto";
+
+package cirq.google.api.v2;
+
+option java_package = "com.google.cirq.google.api.v2";
+option java_outer_classname = "BatchProto";
+option java_multiple_files = true;
+
+// A Batch of multiple circuits that should be run together
+// as one QuantumProgram within Quantum Engine.
+// (Note that circuits may not necessarily be executed
+//  simultaneously on hardware based on circumstances)
```

Comment:
nit: "executed as a batch" instead of "executed simultaneously"

Reply:
I think the wording you proposed adds more confusion.  I rephrased again.